.github/workflows/todo-app-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+on: 
+  schedule:
+    - cron: '25 12 * * 5'
```

Comment:
I can see a testing commit for this and you've potentially already worked out, but this only actually fires on your main branch, hence it won't have fired last week but it looks healthy to me and hopefully should fire this Friday!